### PR TITLE
added e2e integration test to main branch

### PR DESCRIPTION
**Pull request by xinredhat**
rh-pre-commit.version: 2.1.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: .tekton/integration-tests/tasks/rhtap-e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
```
Comment (flacatus) [inline .tekton/integration-tests/tasks/rhtap-e2e.yaml:1]: This task should install rhtap not creating another task for e2e.

Comment (xinredhat) [reply .tekton/integration-tests/tasks/rhtap-e2e.yaml:1]: @flacatus okay, so it should refer to this e2e [task](https://github.com/konflux-qe-incubator/konflux-qe-definitions/blob/main/integration-test/rhtap/rhtap-e2e/task/rhtap-e2e-runner.yaml) , right?

Comment (flacatus) [reply .tekton/integration-tests/tasks/rhtap-e2e.yaml:1]: right!

**Pull request merged**